### PR TITLE
Fixed incubator URLs

### PR DESCRIPTION
**Pull request by jnioche**
Replaces links to 'storm.incubator.apache.org' with 'storm.apache.org' and 'github.com/apache/incubator-storm' with 'github.com/apache/storm'

This also removes the DISCLAIMER file now that the project has become top level.

### CODE REVIEW

File: docs/documentation/Message-passing-implementation.md [hunk 1]
```diff
@@ -8,23 +8,23 @@ documentation: true
...
 - Tasks are responsible for message routing. A tuple is emitted either to a direct stream (where the task id is specified) or a regular stream. In direct streams, the message is only sent if that bolt subscribes to that direct stream. In regular streams, the stream grouping functions are used to determine the task ids to send the tuple to.
-  - Tasks have a routing map from {stream id} -> {component id} -> {stream grouping function} [code](https://github.com/apache/incubator-storm/blob/master/src/clj/backtype/storm/daemon/task.clj#L198)
-  - The "tasks-fn" returns the task ids to send the tuples to for either regular stream emit or direct stream emit [code](https://github.com/apache/incubator-storm/blob/master/src/clj/backtype/storm/daemon/task.clj#L207)
+  - Tasks have a routing map from {stream id} -> {component id} -> {stream grouping function} [code](https://github.com/apache/storm/blob/master/src/clj/backtype/storm/daemon/task.clj#L198)
+  - The "tasks-fn" returns the task ids to send the tuples to for either regular stream emit or direct stream emit [code](https://github.com/apache/storm/blob/master/src/clj/backtype/storm/daemon/task.clj#L207)
```
Comment (kishorvpatil) [inline docs/documentation/Message-passing-implementation.md]: Looks like code references are invalid.

Comment (kishorvpatil) [inline docs/documentation/Message-passing-implementation.md]: Does it mean https://github.com/apache/storm/blob/master/storm-core/src/clj/backtype/storm/daemon/task.clj#L125 ?

File: docs/documentation/Message-passing-implementation.md [hunk 2]
```diff
@@ -8,23 +8,23 @@ documentation: true
...
   - After getting the output task ids, bolts and spouts use the transfer-fn provided by the worker to actually transfer the tuples
-      - Bolt transfer code here: [code](https://github.com/apache/incubator-storm/blob/master/src/clj/backtype/storm/daemon/task.clj#L429)
-      - Spout transfer code here: [code](https://github.com/apache/incubator-storm/blob/master/src/clj/backtype/storm/daemon/task.clj#L329)
+      - Bolt transfer code here: [code](https://github.com/apache/storm/blob/master/src/clj/backtype/storm/daemon/task.clj#L429)
```
Comment (kishorvpatil) [inline docs/documentation/Message-passing-implementation.md]: same as above.

**Comment by kishorvpatil** [comment]
Needs fixing code references to right line numbers. The rest looks great.

**Comment by jnioche** [comment]
thanks for your comments @kishorvpatil 
This particular document is outdated - instead of searching for the relevant line in the master branch I fixed the URLs by pointing them to the branch 0.7.1 which was relevant at the time this document was written.

**Comment by jerrypeng** [comment]
LGTM +1 but probably should update this doc with more current information in the near future

**Comment by kishorvpatil** [comment]
@jnioche Thanks.  +1

**Pull request merged**